testgrid/config/config_test.go

```diff
@@ -86,7 +86,11 @@ func TestConfig(t *testing.T) {
 		}
 
 		// All testgroup from kubernetes must have testgroup name match its bucket name
-		if strings.HasPrefix(testgroup.GcsPrefix, "kubernetes-jenkins/logs/") {
+		// We exclude the postsubmit tests for github.com/tensorflow/k8s because those tests use a different prefix to
+		// be compatible with gubernator.
+		// This issue with tensorflow/k8s might indicate that the test isn't sufficiently general to handle other
+		// repositories.
+		if strings.HasPrefix(testgroup.GcsPrefix, "kubernetes-jenkins/logs/") && !strings.HasPrefix(testgroup.GcsPrefix, "kubernetes-jenkins/logs/tensorflow_k8s") {
 			if strings.TrimPrefix(testgroup.GcsPrefix, "kubernetes-jenkins/logs/") != testgroup.Name {
```

Comment:
I think the check for `kubernetes-jenkins/logs/` is still valid, instead of `TrimPrefix`, we can probably workaround with `filepath.Base(testgroup.GcsPrefix)`